lib/generators/rolemodel/css/base/templates/app/javascript/stylesheets/variables/tokens.scss

```diff
@@ -2,15 +2,24 @@
 // as needed and override them.
 
 @mixin border-radius {
-  -radius-xs:  2px;
+  --radius-xs:  2px;
   --radius-sm:  4px;
   --radius-md:  8px;
-  --radius:     16px;
-  --radius-lg:  24px;
-  --radius-xl:  48px;
+  --radius:     12px;
+  --radius-lg:  16px;
+  --radius-xl:  20px;
   --radius-max: 50%;
 }
 
+@mixin border-width {
+  --border-width: 2px;
+  --border-width-sm: 1px;
+}
+
+@mixin border-strokes {
+  --border-card-outline: 0 0 0 1px var(--color-outline);
```

Comment:
Should this use the `var(--border-width-sm)` instead of 1px?

Reply:
Yes, good catch.